main.go

```diff
@@ -108,30 +109,35 @@ func main() {
 					return
 				}
 
-				// validate the cluster ids contained in the remote write request
-				clusterId, err := remotewrite.ValidateRequest(remoteWriteRequest)
-				if err != nil {
-					log.Printf("error validating remote write request: %v", err)
-					w.WriteHeader(http.StatusForbidden)
-					return
-				}
-
-				log.Println(fmt.Sprintf("remote write request received from %v", clusterId))
+				clusterId := "<unknown>"
```

Comment:
Does this need to be initialized to a non-zero value for some reason?

Reply:
It doesn't have to. It's used in a log later on that could be emitted even for metadata requests. If we leave it empty, then the message would be incomplete. But we could also just not include any cluster id in that message.